intake_stac/catalog.py

```diff
@@ -352,7 +362,7 @@ def _get_driver(self, entry):
 
     def _get_args(self, entry, driver, stacked=False):
         args = entry if stacked else {'urlpath': entry.get('href')}
-        if driver in ['netcdf', 'rasterio', 'xarray_image']:
+        if driver in ['netcdf', 'rasterio', 'xarray_image', 'geopandas', 'parquet']:
```

Comment:
I realize I'm not actually sure what this line does and which drivers should be included, perhaps only the ones that can read in chunks? so maybe not geopandas, not sure about parquet.

Reply:
yeah, I think we can leave this line as it was. Intake/fastparquet should provide reasonable defaults for parquet and there is not a chunked geopandas reader (at this time).